### PR TITLE
PS-8307 Release notes and release note tasks for PS 8.0.29-21

### PR DESCRIPTION
**Pull request by patrickbirch**
Merged 5.7 version
AA source/_res/rn/ps-mysql-blurb.txt
AA source/_res/rn/useful-links.txt
M  source/conf.py
A  source/release-notes/8.0.29-21.rst
M  source/release-notes/release-notes_index.rst

### CODE REVIEW

**Review by alina-derkach-oaza — APPROVED** [review]
LGTM

File: doc/source/release-notes/Percona-Server-5.7.38-41.rst [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+
+Bugs Fixed
+=================================================
+
+* :jirabug:`PS-6029`: Data masking gen_rnd_us_phone() function had a different format compared to MySQL upstream version.
+* :jirabug:`PS-8129`: A fix for when mutex hangs in thread_pool_unix.
+* :jirabug:`PS-8136`: ``LOCK TABLES FOR BACKUP`` did not prevent InnoDB key rotation. Due to this behavior, Percona Xtrabackup couldn't fetch the key in case the key was rotated after starting the backup.
+* :jirabug:`PS-8143`: Fixed the memory leak in ``File_query_log::set_rotated_name()``. 
+* :jirabug:`PS-8204`: When the ``audit_log_format`` was set to XML, logged queries were truncated after a newline character.
+
```
Comment (satya-bodapati) [inline doc/source/release-notes/Percona-Server-5.7.38-41.rst]: we fixed several INSTANT algorithm bugs in 8.0.29, late in the release. I hope they are in pending release state.

**Review by satya-bodapati — CHANGES_REQUESTED** [review]

File: doc/source/release-notes/Percona-Server-5.7.38-41.rst (hunk 1 above)
Comment (satya-bodapati) [reply doc/source/release-notes/Percona-Server-5.7.38-41.rst]: Sorry, I saw them in next commit

File: doc/source/release-notes/8.0.29-21.rst [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+
+Improvements
+=================================================
+
+* :jirabug:`PS-8258`: **Percona Server for MySQL** can be built on the MacOS with RocksDB enabled.
+
+Bugs fixed
+=================================================
+
+* :jirabug:`PS-8921`: An unexpected MySQL 8.0.29 server exit when executing `ALTER TABLE ... DROP COLUMN ALGORITHM=INSTANT`. (Upstream :mysqlbug:`107613`)
```
Comment (satya-bodapati) [inline doc/source/release-notes/8.0.29-21.rst]: @patrickbirch PS-8292 and PS-8303 missing?

File: doc/source/release-notes/8.0.29-21.rst [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+=================================================
+
+Improvements and bug fixes introduced by Oracle for *MySQL* 8.0.29 and included in Percona Server for MySQL are the following:
+
+* The Performance Schema tracks if a query was processed on the PRIMARY engine, InnoDB, or a SECONDARY engine, HeatWave. An EXECUTION_ENGINE column, which indicates the engine used, was added to the Performance Schema statement event tables and the `sys.processlist` and the `sys.x$processlist` views.
+* Added support for the `IF NOT EXISTS` option for the `CREATE FUNCTION`, `CREATE PROCEDURE`, and `CREATE TRIGGER` statements.
+* Added support for `ALTER TABLE ... DROP COLUMN ALGORITHM=INSTANT`. 
+* An anonymous user with the `PROCESS` privilege was unable to select `processlist` table rows.
+
+Find the full list of bug fixes and changes in the `MySQL 8.0.29 Release Notes <https://dev.mysql.com/doc/relnotes/mysql/8.0/en/news-8-0-29.html>`__.
```
Comment (satya-bodapati) [inline doc/source/release-notes/8.0.29-21.rst:27]: @patrickbirch we also have to add a special note about INSTANT corruption and PS choosing INPLACE as default for ADD and DROP columns?

**Review by satya-bodapati — CHANGES_REQUESTED** [review]

File: doc/source/release-notes/8.0.29-21.rst (hunk 3 above)
Comment (patrickbirch) [reply doc/source/release-notes/8.0.29-21.rst:27]: Agreed. I will add the note discussed with @altmannmarcelo when it is available.

File: doc/source/release-notes/8.0.29-21.rst (hunk 2 above)
Comment (patrickbirch) [reply doc/source/release-notes/8.0.29-21.rst]: Added.

**Comment by breisig** [comment]
LGTM

**Review by george-lorch — APPROVED** [review]

**Comment by breisig** [comment]
What is taking this so long to be released? It's been 3 months since 8.0.29 has been released.

File: doc/source/release-notes/8.0.29-21.rst (hunk 2 above)
Comment (yakirgb) [reply doc/source/release-notes/8.0.29-21.rst:43]: PS-8921 does not exist, maybe you meant PS-8291 ?

**Comment by patrickbirch** [comment]
Thank you for your interest. You are welcome to ask questions in the [Percona Community Forum](https://forums.percona.com/c/mysql-mariadb/percona-server-for-mysql-8-0/20)

**Pull request merged**